Cargo.lock

```diff
@@ -2374,6 +2589,83 @@ dependencies = [
  "thiserror",
 ]
 
+[[package]]
+name = "naga"
```

Comment:
Something is obviously wrong here... we shouldn't need `naga` to build `deno`

Reply:
we already do, through webgpu